AUTHORS

```diff
@@ -0,0 +1,27 @@
+# Directly affiliated with DEDIS
+Bryan Ford <bryan.ford@epfl.ch>
+Kokoris Kogias Eleftherios <eleftherios.kokoriskogias@epfl.ch>
+Ismail Khoffi <ismail.khoffi@epfl.ch>
+Linus Gasser <linus.gasser@epfl.ch>
+Nicolas Gailly <nicolas.gailly@epfl.ch>
+Nikitin Kirill <kirill.nikitin@epfl.ch>
+Philipp Jovanovic <philipp.Jovanovic@epfl.ch>
+
+# NON-DEDIS - probably need a CAA or CLA
```

Comment:
I probably wouldn't merge the second part of this file still saying "probably need a CCA or CLA". Also: Aren't David and Patricia automatically covered by this license agreement since they are affiliated with EPFL?